dotCMS/src/integration-test/java/com/dotcms/content/elasticsearch/business/ESReadOnlyMonitorTest.java

```diff
@@ -0,0 +1,315 @@
+package com.dotcms.content.elasticsearch.business;
+
+import com.dotcms.api.system.event.message.MessageSeverity;
+import com.dotcms.api.system.event.message.MessageType;
+import com.dotcms.api.system.event.message.SystemMessageEventUtil;
+import com.dotcms.api.system.event.message.builder.SystemMessageBuilder;
+import com.dotcms.content.elasticsearch.util.RestHighLevelClientProvider;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.business.Role;
+import com.dotmarketing.business.RoleAPI;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.liferay.portal.model.User;
+import com.rainerhahnekamp.sneakythrow.Sneaky;
+import org.elasticsearch.action.admin.cluster.settings.ClusterUpdateSettingsRequest;
+import org.elasticsearch.action.admin.indices.settings.put.UpdateSettingsRequest;
+import org.elasticsearch.action.support.master.AcknowledgedResponse;
+import org.elasticsearch.client.RequestOptions;
+import org.elasticsearch.common.settings.Settings;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+import static com.dotcms.util.CollectionsUtils.list;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.*;
+
+public class ESReadOnlyMonitorTest {
+
+    private static ESReadOnlyMonitor esReadOnlyMonitor;
+    private SystemMessageEventUtil systemMessageEventUtilMock;
+    private RoleAPI roleAPIMock;
+
+    @BeforeClass
+    public static void prepare () throws Exception {
+        //Setting web app environment
+        IntegrationTestInitService.getInstance().init();
+    }
+
+    @Before
+    public void init () {
+        systemMessageEventUtilMock = mock(SystemMessageEventUtil.class);
+        roleAPIMock = mock(RoleAPI.class);
+
+        esReadOnlyMonitor = ESReadOnlyMonitor.getInstance(systemMessageEventUtilMock, roleAPIMock);
+    }
+
+    /**
+     * Method to Test: {@link ESReadOnlyMonitor#start(String)}
+     * When: If the LIVE and WORKING current index are not read only
+     * Should: not sent large message
+     *
+     * @throws DotDataException
+     */
+    @Test
+    public void shouldNotSendLargeMessage() throws DotDataException, IOException {
+        final String message = "message";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "message" appears 5 times in this file; the first occurrence is on line 61](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5642172)